phpstan-baseline.php

```diff
@@ -1151,6 +1166,11 @@
 	'count' => 1,
 	'path' => __DIR__ . '/system/Exceptions/FrameworkException.php',
 ];
+$ignoreErrors[] = [
+	'message' => '#^Method CodeIgniter\\\\Exceptions\\\\FrameworkException\\:\\:forInvalidDirectory\\(\\) has no return type specified\\.$#',
+	'count' => 1,
+	'path' => __DIR__ . '/system/Exceptions/FrameworkException.php',
+];
```

Comment:
@paulbalandan The return type should be `static`, but we cannot use it yet:
https://github.com/codeigniter4/CodeIgniter4/blob/dec4b71ad42488043cb5955dafea22fc5aa3b6d5/system/Exceptions/FrameworkException.php#L36-L39

Reply:
Then a PHPDoc should suffice.